js/views/higher-order/withAutoHide.js

```diff
@@ -55,6 +55,11 @@ const withAutoHide = function(ComposedComponent) {
           this.props.controller.showControlBar();
           this.startHideControlBarTimer();
         }
+
+        if (nextProps.controller.state.cast.connected) {
+          this.props.controller.showControlBar();
```

Comment:
if conditions for prev "if" return true, do we need to showControlBar again?

Reply:
Good catch!!! I will update the logic there to not repeat the same action if the prev and next props has the same value

Thanks,

Reply:
After review the logic behind the method `showControlBar` I found that it handles internally the decision to render or not when it found a difference between the states, so I decided to keep this as is.